README.md

```diff
@@ -40,6 +40,7 @@ Your name | Profile overview | [@yourprofile](https://www.tiktok.com/@yourprofil
 | Hamza  | Random tech                          | [@hamzaelkhoudiri](https://www.tiktok.com/@hamzaelkhoudiri) | [Twitter](https://twitter.com/Hamza87337089)  |
 | Matt   | Coding, tech, and productivity tips! | [@mattupham](https://link.mattupham.com/tiktok)             | [Twitter](https://link.mattupham.com/twitter) |
 | Tim   | Life. Coding, construction, games     | [@timthetinyhomebuilder](https://www.tiktok.com/@timthetinyhomebuilder)| [Insta](https://www.instagram.com/t_alan_v/) |
+| Jon   | Life, Coding, and more random stuff   | [@jonngan](https://www.tiktok.com/@jonngan)| [Insta](https://www.instagram.com/jongan69/) |
```

Comment:
Can you add yourself to the list alphabetically based on the first letter of your TT account? That'd put you after Hamza right now. Thx!